doc/decisions/gsettings_values.md

```diff
@@ -0,0 +1,70 @@
+# GSettings values
+
+## Issue
+
+`GSettings` uses `GLibs` `GVariant` type for it values. The `GVariant` type can hold many
+different variations of types but `Elektra` only distinguishes between strings
+and binary data.
+
+## Constraints
+
+ - saved values have to be readable by `GSettings` but also trough `Elektra` itself
+ - keep `Elektra` and `GLib` as independent as possible
+ - values should be human readable
+ - values should be rather self explaining about their type
+ - saved values should be as portable as possible
+
+## Assumptions
+
+ - `Elektras` strength is its modularity to provide multiple backends that decide how to save values
+ - applications using `GSettings` are well aware about the types they use
+ - `GVariant` types can be represented as strings
+ - `GVariant` types are rather self explaining about their type to some degree e.g. (255, 144) is a tuple of int
+ - `GVariant` string representation can be parsed back into a `GVariant` reliable if we know their type
+ - `GVariant` string representation can be parsed back into a `GVariant` unreliable on type guessing by parsing
+ - `GVariant` can be serialized and deserialized, meaning it is possible to save all types binary
+ - `Elektra` offers types trough templates in its c++ api
```

Comment:
misspelled "through"

Elektra also has type definitions in C (kdbtypes.h), only the type checker (currently) is implemented in C++.